test/TestServerProjectsLowLevel/dpg-customization/Customization/DPGClient.cs

```diff
@@ -73,6 +73,30 @@ public virtual Response<Product> PostModel(string mode, Input input, Cancellatio
             return Response.FromValue((Product)result, result);
         }
 
+        /// <summary> Long running put request that will either return to end users a final payload of a raw body, or a final payload of a model after the SDK has grown up. </summary>
+        /// <param name="waitUntil"> <see cref="WaitUntil.Completed"/> if the method should wait to return until the long-running operation has completed on the service; <see cref="WaitUntil.Started"/> if it should return after starting the operation. For more information on long-running operations, please see <see href="https://github.com/Azure/azure-sdk-for-net/blob/main/sdk/core/Azure.Core/samples/LongRunningOperations.md"> Azure.Core Long-Running Operation samples</see>. </param>
+        /// <param name="mode"> The mode with which you&apos;ll be handling your returned body. &apos;raw&apos; for just dealing with the raw body, and &apos;model&apos; if you are going to convert the raw body to a customized body before returning to users. </param>
+        /// <param name="cancellationToken"> The cancellation token to use. </param>
+        /// <exception cref="ArgumentNullException"> <paramref name="mode"/> is null. </exception>
+        public virtual async Task<Operation<Product>> LroValueAsync(WaitUntil waitUntil, string mode, CancellationToken cancellationToken = default)
+        {
+            var requestContext = new RequestContext {CancellationToken = cancellationToken};
+            var lro = await LroImplementationAsync("DPGClient.LroValue", waitUntil, mode, requestContext);
+            return LowLevelOperationHelpers.Cast(lro, r => (Product)r);
```

Comment:
Why do we use `Cast` for LRO when we used `Select` for Pageable?

Reply:
Because `Select` is applied to items if the page while `Cast` is applied to the operation itself. But I'm open to suggestions